components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -6769,15 +6770,15 @@ def __init__(self, config):
         descriptors = config.getDescriptors(proxy=True)
         self.root = CGList([
             CGGeneric(
-                "#[allow(non_upper_case_globals)]\n" +
-                "pub mod proxy_handlers {\n" +
+                "#[allow(non_upper_case_globals)]\n"
+                "pub mod proxy_handlers {\n"
                 "".join(
```

Comment:
I believe we need a + in between here, otherwise the previous two strings are treated as part of the joining string.